stwcs/updatewcs/astrometry_utils.py

```diff
@@ -96,8 +96,8 @@ def __init__(self, url=None, raise_errors=None, perform_step=True,
              'RAISE_PIPELINE_ERRORS' if set.
 
         perform_step : bool, optional
-            Specify whether or not to perform this step.  This will
-            completely override the setting of the `ASTROMETRY_STEP_CONTROL`
+            Specify whether or not to perform this step.  This is
+            overriden by the setting of the `ASTROMETRY_STEP_CONTROL`
```

Comment:
The behavior is opposite to what this docstring stated, i.e. the env variable always has precedence. I chose to modify the docstring in case someone relies on the behavior.